srcpkgs/speakersafetyd/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'speakersafetyd'
+pkgname=speakersafetyd
+version=1.0.2
+revision=1
+build_style=cargo
+hostmakedepends="alsa-lib pkg-config"
+makedepends="alsa-lib-devel"
+short_desc="Software Smart Amp implementation"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
new packages shouldn't be orphaned (same with mesa-asahi)

Reply:
done (x2)

Reply:
others too.